CRM/Core/Resources.php

```diff
@@ -651,7 +651,9 @@ public function addCoreStyles($region = 'html-header') {
         $this->addStyleUrl($customCSSURL, 99, $region);
       }
       if (!Civi::settings()->get('disable_core_css')) {
-        $this->addStyleFile('civicrm', 'css/civicrm.css', -99, $region);
+        if (!$config->userFrameworkFrontend || ($config->userFrameworkFrontend && !Civi::settings()->get('disable_frontend_core_css'))) {
+          $this->addStyleFile('civicrm', 'css/civicrm.css', -99, $region);
```

Comment:
@agilewarealok how would this work in Drupal because i think `$config->userFrameworkFrontend` is not a thing in Drupal implementation so it feels like for Drupal it might be both `disable_core_css` and `disable_frontend_core_css` behave the same way

Reply:
Then maybe userFrameworkFrontend should be added to the Drupal implementation, since Drupal has an API available, https://api.drupal.org/api/drupal/includes%21path.inc/function/path_is_admin/7.x

So would make sense to add it.

Reply:
So if two other PRs are submitted, to cater for Joomla and Drupal - userFrameworkFrontend. Then we can get the trio reviewed together. Hows' that for a plan?